eva-accession-clustering-automation/clustering_automation/create_clustering_progress_table.py

```diff
@@ -70,218 +69,213 @@ def create_table(private_config_xml_file):
+                                         mongo_source):
+    for asm_acc in asm_sources:
+        fill_should_be_released_for_taxonomy_and_assembly(private_config_xml_file, tax, asm_acc, asm_sources[asm_acc],
+                                                          profile, release_version, mongo_source)
+
+
+def fill_should_be_released_for_all_in_release(private_config_xml_file, profile, release_version, mongo_source):
+    tax_asm = get_taxonomy_list_for_release(private_config_xml_file, profile, release_version)
+    for tax in tax_asm:
+        fill_should_be_released_for_taxonomy(private_config_xml_file, tax, tax_asm[tax], profile, release_version,
+                                             mongo_source)
 
 
 def main():
     parser = argparse.ArgumentParser(description='Create and load the clustering and release tracking table',
                                      add_help=False)
-    parser.add_argument("--private-config-xml-file", help="ex: /path/to/eva-maven-settings.xml", required=True)
-    parser.add_argument("--release-version", help="version of the release", type=int, required=True)
-    parser.add_argument("--reference-directory",
-                        help="Directory where the reference genomes exists or should be downloaded", required=True)
-    parser.add_argument("--taxonomy", help="taxonomy id for which rs count needs to be updated", type=int,
-                        required=False)
-    parser.add_argument('--tasks', required=False, type=str, nargs='+', default=all_tasks, choices=all_tasks,
-                        help='Task or set of tasks to perform.')
+    parser.add_argument("--private-config-xml-file", required=True, help="ex: /path/to/eva-maven-settings.xml")
+    parser.add_argument("--release-version", required=True, type=int, help="version of the release")
+    parser.add_argument("--reference-directory", required=True,
+                        help="Directory where the reference genomes exists or should be downloaded")
+    parser.add_argument("--profile", required=True, help="profile where entries should be made e.g. development")
+    parser.add_argument('--tasks', required=False, type=str, nargs='+', choices=all_tasks,
+                        help='Task or set of tasks to perform')
+    parser.add_argument("--taxonomy", required=False, type=int,
+                        help="taxonomy id for which should be released needs to be updated")
+    parser.add_argument("--assembly", required=False,
+                        help="assembly accession for which should be released needs to be updated")
     parser.add_argument('--help', action='help', help='Show this help message and exit')
     args = parser.parse_args()
 
     logging_config.add_stdout_handler()
 
     if not args.tasks:
-        args.tasks = all_tasks
+        args.tasks = ['fill_release_entries']
+
+    if 'create_table' in args.tasks:
+        create_table_if_not_exists(args.private_config_xml_file)
```

Comment:
I'm not sure there a point in creating a task for the table creation. 
I would remove the task altogether and always run `create_table_if_not_exists`.

Reply:
removed